docs/blog/2021-09-27-crash.md

```diff
@@ -0,0 +1,328 @@
+---
+title: 50 ways to crash our product
+author: Andre Bogus 
+author_title: Chief Rustacean
+author_url: https://github.com/getsynth
+author_image_url: https://avatars.githubusercontent.com/u/4200835?v=4
+tags: [synth, testing, story]
+description: Trying to break our own code for fun and profit
+image: https://commons.wikimedia.org/wiki/File:Dialog-error-round.svg#/media/File:Dialog-error-round.svg
+hide_table_of_contents: false
+---
```

Comment:
I would add an image at the top to a: make the content more engaging, and b: better thumbnail for the link. Maybe something like this -> 
![image](https://user-images.githubusercontent.com/14791384/135275893-f8946458-871a-4751-96fa-25f29a978e91.png)
?